### PR TITLE
Google Play Publishing to custom track

### PR DESCRIPTION
**Pull request by priitlatt**

### CODE REVIEW

File: content/publishing-yaml/distribution.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ publishing:
 
 ### Google Play
 
-Codemagic enables you to automatically publish your app to the `internal`, `alpha`, `beta` and `production` tracks on Google Play. In order to do so, you will need to set up a service account in Google Play Console and add the `JSON` key file to your Codemagic configuration file, see how to [set up a service account](../knowledge-base/google-play-api/). The proper way to add your keys to `codemagic.yaml` is to [encrypt](../building/encrypting) the contents of the key file and add the encrypted value into the configuration file.
+Codemagic enables you to automatically publish your app either to one of the predefined tracks (`internal`, `alpha`, `beta` and `production`) on Google Play, or to your custom closed testing tracks. In order to do so, you will need to set up a service account in Google Play Console and add the `JSON` key file to your Codemagic configuration file, see how to [set up a service account](../knowledge-base/google-play-api/). The proper way to add your keys to `codemagic.yaml` is to [encrypt](../building/encrypting) the contents of the key file and add the encrypted value into the configuration file.
```
Comment (helinanever) [inline content/publishing-yaml/distribution.md:73]: should we mention internal app sharing in this list?

File: content/publishing/publishing-to-google-play.md [hunk 2]
```diff
@@ -34,7 +34,9 @@ The very first version of the app must be added to Google Play manually. You can
    - Alpha --- publish for testing with a small group of trusted users
    - Beta --- publish for testing to a wider set of users
    - Production --- release the app to production
-5. If your application supports [in-app updates](https://developer.android.com/guide/playcore/in-app-updates), set the desired priority. If not, select **Default**.
-6. In case you want to release a [staged version](https://support.google.com/googleplay/android-developer/answer/6346149?hl=en) of your application, which reaches only a fraction of users, set **Rollout fraction** to a value between 0 and 1. To release to everyone, leave the value empty.
-7. If you want to publish the .apk even when one or more tests fail, mark the **Publish even if tests fail** checkbox.
-8. Select **Enable Google Play publishing** at the top of the section to enable publishing.
+   - Custom --- release the app to custom closed testing track
```
Comment (helinanever) [inline content/publishing/publishing-to-google-play.md:37]: ... to a custom...

File: content/publishing/publishing-to-google-play.md [hunk 3]
```diff
@@ -34,7 +34,9 @@ The very first version of the app must be added to Google Play manually. You can
...
+5. If you want to publish to custom track, enter the custom track name.
```
Comment (helinanever) [inline content/publishing/publishing-to-google-play.md:38]: ... to a custom..

File: content/publishing/publishing-to-google-play.md (hunk 2 above)
Comment (priitlatt) [reply content/publishing/publishing-to-google-play.md:37]:
```suggestion
   - Custom --- release the app to a custom closed testing track
```

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**